students/juyeob/westagram/users/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
+    name = models.CharField(max_length=45)
+    email = models.EmailField(max_length=45)
+    password = models.CharField(max_length=45)
+    phone = models.CharField(max_length=45)
+    class Meta:
+        db_table = 'users'
```

Comment:
- 변수명은 의미를 정확히 해주세요.
- 변수명과 class 사이에 공백 추가해 주세요.
- email field에 unique 속성을 추가하셔서 유저가 중복 저장되는 문제가 발생하지 않도록 수정해주세요!(공식 문서에서 그 방법을 찾아보고 적용해주세요.)
- 이름과 관련된 필드는 래를 참고해서 수정해 주세요.
    - id(계정) → `username`
    - name → `first_name`, `last_name`
 - =을 기준으로 정렬해 주세요.
 
```suggestion
class User(models.Model):
    name         = models.CharField(max_length=45)
    email        = models.EmailField(max_length=45)
    password     = models.CharField(max_length=45)
    phone_number = models.CharField(max_length=45)
    
    class Meta:
        db_table = 'users'
```